umh.docs.umh.app/content/en/docs/getstarted/movingtoproduction.md

```diff
@@ -54,8 +51,10 @@ The installation is well documented in the [first chapter](/docs/getstarted/inst
 #### 4. Network Configuration
 
 Once your UMH instance is up and running, ensure it is placed in the same network as your machines. Additionally, ensure that the device running the Management Console is also within the same network.
+This step is crucial for facilitating seamless communication, especially when using advanced tools like Grafana or Node-RED.
```

Comment:
what do you mean with facilitating seamless communication? wht is Grafana and Node-RED advanced?

Reply:
To be honest I think we could just get rid of this specific line.